tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -2936,6 +2936,7 @@ public void TestMapReduceWriteConcern()
         }
 
         [Fact]
+        [Obsolete("ParallelScan was deprecated in version 4.1.")]
```

Comment:
Good instinct in tweaking the ParallelCollectionScan tests as well so that we don't get an warning on compile :) (As an aside, the driver project treats all warnings as errors due to the `<TreatWarningsAsErrors>` element in the .csproj files.) The driver has taken a slightly different approach though: typically, to get around warnings, the driver use a a pair of `#pragma` statements with zero indentation to temporarily disable the obsolete warning while testing, like so:

```
#pragma warning disable 618
                var enumerators = _collection.ParallelScanAs(typeof(BsonDocument), new ParallelScanArgs
                {
                    BatchSize = 100,
                    NumberOfCursors = numberOfCursors
                });
#pragma warning restore
```